src/Components/Web/src/Forms/ValidationSummary.cs

```diff
@@ -73,7 +78,7 @@ protected override void BuildRenderTree(RenderTreeBuilder builder)
                 first = false;
 
                 builder.OpenElement(0, "ul");
-                builder.AddAttribute(1, "class", "validation-errors");
+                builder.AddAttribute(1, "class", string.IsNullOrWhiteSpace(CssClass) ? "validation-errors" : $"validation-errors {CssClass}");
```

Comment:
As I state [here](https://github.com/dotnet/aspnetcore/issues/43860#issuecomment-1244145778), other components seem to have `CssClass` as a get only property and use `AttributeUtilities.CombineClassNames` in `OnParametersSet` to append the class name passed in via `AdditionalAttributes`. Let me know if I should update this code to do that as well.

Thanks! :)

Reply:
Yes, sorry, you're right. It would be better to align more with the pattern we have elsewhere of extracting the `class` value from `AdditionalAttributes`, with `CssClass` being a readonly property that combines the generated class with any supplied by the developer. Sorry for the misleading advice previously.